src/Project.js

```diff
@@ -554,13 +555,18 @@ class Project extends DirItem {
      */
     run() {
         let exitValue = 0;
-        const results = this.findIssues(this.options.locales);
+        const results = this.findIssues(this.options.opt.locales);
         this.resultStats = {
             errors: 0,
             warnings: 0,
             suggestions: 0
         };
 
+        // make sure the results are organized by the order the lines appear in the
+        // source file in order to make it easier for the engineer to fix all the
+        // problems in the source file sequentially.
+        results.sort(ResultComparator);
```

Comment:
Nitpick: shouldn't this be placed within `if (results)`? Not that it matters much because by definition `this.findIssues()` should return an array anyway